demo/pom.xml

```diff
@@ -53,14 +53,33 @@
         <dependency>
             <groupId>dk.digst</groupId>
             <artifactId>oiosaml3.java</artifactId>
-            <version>3.0.3-SNAPSHOT</version>
+            <version>3.1.0-SNAPSHOT</version>
         </dependency>
 
+        <!-- Write all logging to Log4j2 -->
         <dependency>
-            <groupId>commons-logging</groupId>
-            <artifactId>commons-logging</artifactId>
-            <version>1.2</version>
-            <scope>compile</scope>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-slf4j-impl</artifactId>
+            <version>2.14.1</version>
```

Comment:
Maybe in the light of https://cve.mitre.org/cgi-bin/cvename.cgi?name=2021-44228 we should upgrade to https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-slf4j-impl/2.15.0